common-lib/src/main/scala/models/Stub.scala

```diff
@@ -46,7 +46,9 @@ case class ExternalData(
                          actualNewspaperPublicationDate: Option[LocalDate] = None,
                          // Description enriched for use by WF front end client code.
                          shortActualPrintLocationDescription: Option[String] = None,
-                         longActualPrintLocationDescription: Option[String] = None) {
+                         longActualPrintLocationDescription: Option[String] = None,
+                         statusInPrint: Option[String] = None,
```

Comment:
```suggestion
                         statusInPrint: Status = Status.Writers,
```

I'm suggesting in https://github.com/guardian/workflow/pull/1070/files#diff-f5d71e0998bbbbf694aebeb923220c87 that we make this match the existing `status` field in terms of it's type.